cmd/containerd-shim-runhcs-v1/serve.go

```diff
@@ -14,7 +14,7 @@ import (
 
 	"github.com/Microsoft/go-winio"
 	task "github.com/containerd/containerd/api/runtime/task/v2"
-	"github.com/containerd/containerd/log"
```

Comment:
Is there a reason why we use `containerd/log` here instead of `internal/log`? (I think this is more of a question for @kevpar)

Reply:
That's fair. @dmcgowan would you mind removing this import altogether, and changing the `log.RFC3339NanoFixed` in this file to `hcslog.TimeFormat`?

Reply:
Updated. The test package is still using `github.com/containerd/log`, it seems isolated but maybe it could just use logrus directly in the future.

Reply:
Agreed, we can try to move that in the future